boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp.yaml

```diff
@@ -10,7 +10,7 @@ toolchain:
   - xtools
   - zephyr
 ram: 188
-flash: 324
+flash: 1428
```

Comment:
Discussed with @barsok, we should provide only application partition size here (not whole available flash). Leaving as is.